src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/simple_response_commands_test.cc

```diff
@@ -131,6 +127,35 @@ TYPED_TEST(MobileResponseCommandsTest, Run_SendResponseToMobile_SUCCESS) {
   command->Run();
 }
 
+template <class CammandWithUnsubscribe>
```

Comment:
Typo C**o**mmandWithUnsubscribe

Reply:
@APipko Fixed in return value bool returned, because we need that parameter for Unit tests..